scripts/publish-storybook-temp.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
Yeah, this file is there so I don't lose it. At some point, we're going to use Conductor for our dev Storybook instead of direct S3 if I can accomplish it.